version/version.go

```diff
@@ -17,6 +17,6 @@
 package version
 
 var (
-	Version       = "1.2.0"
-	DriverVersion = "1.2.0"
+	Version       = "1.3.0"
+	DriverVersion = "1.3.0"
```

Comment:
what the diff?

Reply:
what do you mean?

Reply:
version vs DriverVersion

Reply:
[your guess is as good as mine](https://github.com/IBM/ibm-block-csi-operator/pull/76/files#r438116211).
`DriverVersion` sounds pretty straightforward, and I see `Version` usage [here](https://github.com/IBM/ibm-block-csi-operator/blob/aa140208f60135ad1e7819bb85ce53bfe63d8f8d/pkg/internal/ibmblockcsi/ibmblockcsi.go#L72)

Reply:
so Driver is controller an Node? like in the github?

Reply:
ok, sounds right, if we will have patches it will be handy to separate it..